### DISCUSSION TITLE
Multiple aggregations

### DISCUSSION THREAD

**Discussion by vitolpoc** [infinite-table/infinite-react#19] [Q&A] [answered]
Is it possible to configure the table to accept multiple aggregations ?

In the following sandbox, starting from the documentation example, I defined a new column aggregation in the `columnAggregations` Map on the age field. As a result (and with probably more configuration) I would expect to have one column by aggregation (salary and age) under each pivot column.
The first column hierarchies would be:
- India > yes > average salary
- India >yes > average age
- India > no > average salary
- India >no > average age
- India > total > average salary
- India >total > average age
- Mexico > no > average salary
- Mexico >no > average age
- ...

https://codesandbox.io/s/infinite-table-pivot-multiple-aggregations-5bcni?file=/src/App.tsx

**Comment by roblotter** [comment, marked as answer]
Shipped in `0.1.0`

See https://codesandbox.io/s/infinite-table-pivot-multiple-aggregations-forked-i2zj2?file=/src/App.tsx
